init.lua

```diff
@@ -258,6 +258,10 @@ minetest.register_node("anvil:anvil", {
 		end
 
 		local player_name = player:get_player_name()
+		local owner = meta:get_string("owner")
```

Comment:
Suggestion: Move this `have_access`-style check into a local helper function so that shared anvils are properly handled everywhere.

Reply:
Done in this [commit](https://github.com/minetest-mods/anvil/pull/35/commits/5d0393ba56ac426585de4470a004d2b6017bbff2)